src/cephadm/cephadm

```diff
@@ -2615,6 +2615,9 @@ def command_bootstrap():
         logger.info('Setting orchestrator backend to cephadm...')
         cli(['orch', 'set', 'backend', 'cephadm'])
 
+        if args.ssh_user != 'root':
+            cli(['cephadm', 'set-user', args.ssh_user])
```

Comment:
```suggestion
        cli(['cephadm', 'set-user', args.ssh_user])
```

maybe bootstrap will be re-run using `root` as the ssh-user (e.g. bootstrap fails after this point)

Reply:
We're now encountering the fact that people download cephadm form the source tree.

**We can't do that this way.**

As soon as we merge this into octopus, people will download the new cephadm binary, but still using the old MGR module. You'' need to do this in a backward compatible way. Something like

```bash
ceph config-key mgr/cephadm/ssh_user $ssh_user
```